packages/website/src/building-blocs/getCodeSandboxLink.tsx

```diff
@@ -0,0 +1,66 @@
+import {getParameters} from 'codesandbox/lib/api/define';
+
+const packageJson = {
+    dependencies: {
+        react: 'latest',
+        'react-dom': 'latest',
+        '@coveord/plasma-mantine': 'latest',
+        '@emotion/react': 'latest',
+        '@mantine/core': 'latest',
+        '@mantine/dates': 'latest',
```

Comment:
Would it be possible to have the same version as in the package.json? To avoid cases where our component is fine at version X.x.x but is now broken in the latest version because of mysterious reasons 🤔 (looking at you ModalWizard on 5.7.0 😐 )...

Reply:
I'll see what I can do!

Reply:
I'll do it in a separate PR